spec/requests/rich_text_reactions_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'RichTextReaction requests' do
+  let!(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  describe 'POST /rich_text_reactions' do
+    it 'creates a rich text reaction' do
+      expect {
+        post rich_text_reactions_url,
+          params: {
+            rich_text_reaction: {
+              emoji_caption: RichTextReaction.emoji_captions.first,
+              rich_text_id: create(:standup_meeting).yesterday_work_description.id
+            },
+            format: :turbo_stream
+          }
+      }.to change(RichTextReaction, :count).by 1
+      expect(response).to have_http_status :created
+    end
+
+    it 'sets a flash message to indicate failure to create' do
+      post rich_text_reactions_url,
+        params: {
+          rich_text_reaction: {
+            emoji_caption: RichTextReaction.emoji_captions.first,
+            rich_text_id: 2
+          },
+          format: :turbo_stream
+        }
+
+      expect(flash.now[:alert]).to be_present
+    end
+  end
+
+  describe 'DELETE /rich_text_reactions/:id' do
+    it 'deletes a rich text reaction' do
+      id = create(:rich_text_reaction).id
+
+      expect {
```

Comment:
Same comment as above regarding the location of the expectation.

Reply:
Same question as [this one](https://github.com/agency-of-learning/PairApp/pull/451#discussion_r1435717429).